_includes/configuration.md

```diff
@@ -1,152 +1,47 @@
 ## Configuring {{ site.data.product.title_short }}
 
-After installing {{ site.data.product.title_short }} and running it for the first
-time, you must perform some basic configuration. To configure
-{{ site.data.product.title_short }}, you must at a minimum:
+After you install {{ site.data.product.title_short }} and run it for the first time, you must perform some basic configuration. To configure {{ site.data.product.title_short }}, you must complete these steps:
 
-1.  Add a disk to the infrastructure hosting your appliance.
+- Add a disk to the infrastructure that is hosting your appliance.
 
-2.  Configure the database.
-
-Configure the {{ site.data.product.title_short }} appliance using the internal
-appliance console.
+Configure the {{ site.data.product.title_short }} appliance by using the internal appliance console.
 
 ### Accessing the Appliance Console
 
-1.  Start the appliance and open a terminal console.
+1. Start the appliance and open a terminal console.
 
-2.  Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance
-    summary screen displays.
+2. Enter the `appliance_console` command. The {{ site.data.product.title_short }} appliance summary screen displays.
 
-3.  Press `Enter` to manually configure settings.
+3. Press `Enter` to manually configure settings.
 
-4.  Press the number for the item you want to change, and press `Enter`.
-    The options for your selection are displayed.
+4. Press the number for the item you want to change, and press `Enter`. The options for your selection are displayed.
 
-5.  Follow the prompts to make the changes.
+5. Follow the prompts to make the changes.
 
-6.  Press `Enter` to accept a setting where applicable.
+6. Press `Enter` to accept a setting where applicable.
 
 **Note:**
 
-The {{ site.data.product.title_short }} appliance console automatically logs out
-after five minutes of inactivity.
-
-### Configuring a Database
-
-{{ site.data.product.title_short }} uses a database to store information about the
-environment. Before using {{ site.data.product.title_short }}, configure the database
-options for it; {{ site.data.product.title_short }} provides the following two
-options for database configuration:
-
-  - Install an internal PostgreSQL database to the appliance
-
-  - Configure the appliance to use an external PostgreSQL database
-
-#### Configuring an Internal Database
-
-{% include configuration-db.md %}
-
-#### Configuring an External Database
```

Comment:
This should stay...we still support external databases.  It's only the internal database that's technically wrong because we come with one out of the box, but the instructions say otherwise.